dist/mapdc.css

```diff
@@ -425,6 +425,14 @@ body {
   padding: 0 4px;
   text-decoration: none; }
 
+.basemap-satellite .gradient-legend .block .text, .basemap-dark .gradient-legend .block .text, .basemap-odyssey .gradient-legend .block .text, .basemap-decimal .gradient-legend .block .text {
```

Comment:
@uyanga-gb it might be good to dismiss these changes and ask @menaczar to add them in a separate PR as they don't relate to this PR. They are from: https://github.com/mapd/mapd-charting/pull/197/files

Reply:
@clhenrick, sure I will do that.

Reply:
Here is the PR https://github.com/mapd/mapd-charting/pull/206
@uyanga-gb feel free to merge it